### DISCUSSION TITLE
Matter ACL / `Acl` attribute of `AccessControl` Cluster on Endpoint 0 ClusterId 31 (0x001f) is missing

### DISCUSSION THREAD

**Discussion by lboue** [arendst/Tasmota#22571] [Support request]
`Acl` attribute of `AccessControl` Cluster on Endpoint 0 ClusterId 31 (0x001f) is missing

**Tastoma**

![image](https://github.com/user-attachments/assets/a92a22f7-c8f9-4086-8776-380ca3ab70ca)

**Eve Energy**

![image](https://github.com/user-attachments/assets/0570def8-412c-4ab1-a4f8-bd6d0246a9df)

Ref 
24-27349-006_Matter-1.4-Core-Specification.pdf page 391

Is is possible to fix it?

Regards

**Comment by s-hadinger** [comment]
There is no plan to implement ACL in Tasmota Matter. There was no use up to now and I want to implement the minimum to avoid anything to go wrong with coordinators.

Is it a blocker?

**Comment by lboue** [comment]
My goal is to test Matter bindings with Tasmota:
1) Write the Acl to Tasmota Matter light
2) Set up the binding (Matter switch device)

Then remote control the light with the Matter switch.

**Reply by s-hadinger** [reply]
Neither ACL nor bindings are supported. Sorry
